src/mjolnir/shortcutbuilder.cc

```diff
@@ -432,7 +435,8 @@ uint32_t AddShortcutEdges(GraphReader& reader,
       // Connect edges to the shortcut while the end node is marked as
       // contracted (contains edge pairs in the shortcut info).
       uint32_t rst = 0;
-      uint32_t opp_local_idx = 0;
+      // For turn duration calculation during contraction
+      uint32_t opp_local_idx = directededge->opp_local_idx();
```

Comment:
this isnt actually needed right? it would be just as fine to set it to `-1` right?

Reply:
Not getting the idea 🤔

Reply:
sorry i misread the code!